collect_app/src/test/java/org/odk/collect/android/widgets/AudioWidgetTest.java

```diff
@@ -98,6 +102,12 @@ public void setUp() {
         when(activityAvailability.isActivityAvailable(any())).thenReturn(true);
     }
 
+    @After
+    public void tearDown() {
+        widgetActivity.originalFiles.clear();
```

Comment:
You shouldn't need to do this as each test will get a new instance of `TestWidgetActivity`.